L30-cpp-server-load-reporting.md

```diff
@@ -0,0 +1,42 @@
+﻿C++ API changes for Server Load Reporting
+----
+* Author(s): AspirinSJL
+* Approver: markdroth, vjpai
+* Status: Draft
+* Implemented in: C++
+* Last updated: 2018-07-13
+* Discussion at: TBA
+
+## Abstract
+
+As part of the load balancing functionality, the server-side load reporting service is to be introduced. A `LoadReportingServiceServerBuilderOption` will be used to enable that service. A helper function `AddLoadReportingCost()` can be used to report the customized call metrics.
+
+## Background
+
+A load balancer needs to get the load data of the servers in order to balance the load among the servers accordingly. One possible approach to get the load data is to request the load data from the servers via RPCs. That requires the servers to register an additional service to report the load data.
+
+
+### Related Proposals: 
+N/A
+
+## Proposal
+
+We propose a new public header `include/grpcpp/ext/server_load_reporting.h` to use the server load reporting service. Note that this public header (and the load reporting service) is only available when the binary is built with the `grpcpp_server_load_reporting` library. That library is only available in Bazel build because it depends on OpenCensus which can only be built with Bazel.
+
```

Comment:
Will the new public header depend on anything about OpenCensus, or will it only be the implementation? If it's the header itself, we are likely to have another set of issues (related to grpc/grpc#15984, an Abseil dependence, and other issues)

Reply:
It's only the implementation that depend on OpenCensus; the public header doesn't. So we should be safe!